### ISSUE TITLE
Usability: NVDA is not reciting the name for the buttons  and files  available in the debug sidebar area.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/vscode#17240]
VS Code Version: 1.6.1
OS Version:1607
NVDA version:-2016.2.1

**Steps to Reproduce:**
Prerequisite: Node.js and git repository need to be installed.
1.Open visual studio code.
2.Using "Ctrl+Shift+D", key navigate to the Git button.
3.Using "Tab", key navigate to the "variables","watch","callstack" and "Breakpoint" section.

**Expected result:**
NVDA should recite the name for the buttons and files present in the "variables","watch","callstack" and "Breakpoint" section.

**Actual Result:**
NVDA is not reciting the name for the buttons and files present in the "variables","watch","callstack" and "Breakpoint" section.
**User Impact:**
Screen Reader user doesn't get to know the purpose of the control as it's not getting announced

**Attachments:**
![inspectinformation_visualstudiocodeeditor_debugsidebar](https://cloud.githubusercontent.com/assets/22956130/21213230/085bfc90-c2b9-11e6-9b24-aca934420636.jpg)
![narratorbuddyinformation_vscodeeditor_debugsidebar](https://cloud.githubusercontent.com/assets/22956130/21213231/085c668a-c2b9-11e6-8a83-b7cd9db73e2a.jpg)
![nvda_speechviewerinformation_vscodeeditor_debugsidebararea](https://cloud.githubusercontent.com/assets/22956130/21213232/085ed438-c2b9-11e6-9239-de906a9d3239.jpg)

**Comment by isidorn** [comment]
This works fine for me as shown in the attached gif
@WiproPSAccTest the interaction which you might not have done is using the up / down key to navigate through each tree per section. Since tab will jump to the other section - this makes navigating faster and easier imho.
Please let me know if I am missing something here and we can reopen

![nvda](https://cloud.githubusercontent.com/assets/1926584/21348202/463c3934-c6ac-11e6-87f7-cb398e1789b7.gif)

**Comment by WiproPSAccTest** [comment]
@isidorn The issue is not reproducing in vs code editor 1.8.1. The issue is fixed now.

**Comment by isidorn** [comment]
@WiproPSAccTest thanks for checking!
